--- ./mysql-4.0.27/mysys/my_thr_init.c.old	2006-05-07 03:55:48.000000000 +0900
+++ ./mysql-4.0.27/mysys/my_thr_init.c	2019-01-18 15:20:49.000000000 +0900
@@ -37,9 +37,9 @@ pthread_mutex_t LOCK_localtime_r;
 #ifndef HAVE_GETHOSTBYNAME_R
 pthread_mutex_t LOCK_gethostbyname_r;
 #endif
-#ifdef PTHREAD_ADAPTIVE_MUTEX_INITIALIZER_NP
+//#ifdef PTHREAD_ADAPTIVE_MUTEX_INITIALIZER_NP
 pthread_mutexattr_t my_fast_mutexattr;
-#endif
+//#endif
 #ifdef PTHREAD_ERRORCHECK_MUTEX_INITIALIZER_NP
 pthread_mutexattr_t my_errchk_mutexattr;
 #endif
